src/composables/useSearchParams.ts

```diff
@@ -7,10 +7,6 @@ type Facet = {
   value: string;
 };
 
-const facets = ref<Facet[]>([]);
-const query = ref<string>('');
-const search = ref('');
-
```

Comment:
Vite may transform modules by bundling them into IIFEs but not sure

Reply:
I can't reset them on the tests because I'm testing a component that uses the composable and because they are tightly coupled, mocking the composable will make the tests useless, but I can reset the Modules using scoped imports so the variables are not cached between tests.

Reply:
Curious about that approach but was thinking of just resetting the global state to the initialized values in between tests.